### ISSUE TITLE
[AUTOMATION]: Findings from Nightwatch accessibility tests

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#21780]
## User Story or Problem Statement
A recent staging review of the Eastern Oklahoma VAMC (VA Medical Center) revealed a few accessibility tests were failing in the axe browser plugin and axe-core command line interface (CLI) but not logging errors in our Nightwatch static content axe checks. Further investigation has identified some additional tests failing silently.

This epic will collect the type and location of these tests failing silently so we can triage and fix them.
